spinach/task.py

```diff
@@ -218,7 +218,7 @@ def schedule(self, task: Schedulable, *args, **kwargs):
         Spinach :class:`Engine`.
         """
         self._require_attached_tasks()
-        self._spin.schedule(task, *args, **kwargs)
+        return self._spin.schedule(task, *args, **kwargs)
 
```

Comment:
This method signature and docstring should also be updated to reflect the new return value.